VAST/h/VType.h

```diff
@@ -4,12 +4,15 @@
 #include <string>

```

Comment:
Overall, very good... would suggest writing pass/fail criteria/descriptions for each test listed

Reply:
good idea